frontend/src/static/js/v_segment.js

```diff
@@ -1,3 +1,21 @@
+window.toggleSegment = function toggleSegment(ele) {
+  // function for toggling unopened segment
+  const targetClass = 'active toggled';
```

Comment:
The `toggled` is needed to separate between untouched code that are combined with the touched code and untouched code that needs to be toggled in order to view them.

Reply:
Sorry, I do not get your point. Do you mean the untouched code with less than 4 lines? Can you give an example?

Reply:
> Do you mean the untouched code with less than 4 lines? 

Yes.

> Can you give an example?

The red boxes are untouched code needs to be toggled whereas the blue box are untouched code that combines with the touched code since it is less than 4 lines.
<img width="794" alt="Screen Shot 2019-06-29 at 6 16 46 PM" src="https://user-images.githubusercontent.com/32864116/60382778-a2e3f400-9a9a-11e9-82be-959bb5b4434a.png">

Reply:
Let me reconsider my solution, the `toggled` class may not be necessary. Sorry for the inconvenience caused.

Reply:
The` toggled` class is for the button only, however, the button can directly watch `active` class.

Reply:
Yes you are right. I have made changes to the code.